src/main/java/com/mojang/datafixers/types/JsonOps.java

```diff
@@ -71,8 +71,12 @@ public JsonElement empty() {
 
     @Override
     public Optional<Number> getNumberValue(final JsonElement input) {
-        if (input.isJsonPrimitive() && input.getAsJsonPrimitive().isNumber()) {
-            return Optional.of(input.getAsNumber());
+        if (input.isJsonPrimitive()) {
```

Comment:
I'd question why there isn't a `Optional<Boolean> getBooleanValue(final JsonElement input);`

Reply:
@Sollace look at https://github.com/Mojang/DataFixerUpper/pull/42/files#diff-f530d22829ca7397524184c8038357d4R90

Reply:
Oh. Huh.

That didn't show on the diff. :/

Reply:
That might be because I added it in a second commit.